app/src/main/java/org/jellyfin/mobile/bridge/NativeInterface.kt

```diff
@@ -60,9 +61,7 @@ class NativeInterface(private val fragment: WebViewFragment) : KoinComponent {
             // normalize the name by removing special characters
             // and making sure it's at least 1 character long
             // otherwise the webui will fail to send it to the server
-            val name = deviceInfo.name
-                .replace("[^\\x20-\\x7E]".toRegex(), "")
-                .trim()
+            val name = AuthorizationHeaderBuilder.encodeParameterValue(deviceInfo.name)
                 .padStart(1)
```

Comment:
Could probably move this into a single line ¯\\\_(ツ)\_/¯
```suggestion
            val name = AuthorizationHeaderBuilder.encodeParameterValue(deviceInfo.name).padStart(1)
```